### ISSUE TITLE
Accessibility - Video player - screen reader & iPhone - Video player frame title

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1774]
## Detailed description

Issue :- 
Screen reader is announcing this frame Title as "Cultural Dynamic Video player frame :". Here, instead of providing the video player name as the title, A concise description of the video content can be given. This would be more useful for the user. 
In iPhone screen reader is not announcing the frame title.

Expected - 
Provide the following text as the frame title. 
"A dated advertisement for Folgers Coffee (1:00)"

**Comment by RobertaJHahn** [comment]
Moving this issue to sprint 20-08.  The dev work has to be done during the first week of the sprint to include it in release v1.6.0.

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 
We just got an update regarding this merged in, could you please check this again?
Thank you!

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu  Re tested this, screen reader (JAWS) not fixed: Can you provide a brief title in title attribute for iframe describing the video content.
Video player - screen reader & iPhone - Video player frame title not announced.

**Comment by RaphaelAmadeu** [comment]
This was one of the issues that we can't solve on our side,
as this on kaltura's end, I think we can close this,
Thank you!
CC: @RobertaJHahn , @ljcarot , @jeffchew

**Comment by ljcarot** [comment]
ok, I'll close. thx @RaphaelAmadeu
